desktop/mac/install.md

```diff
@@ -94,7 +94,9 @@ Your Mac must meet the following requirements to install Docker Desktop successf
 
 After downloading `Docker.dmg`, run the following commands in a terminal to install Docker Desktop in the Applications folder:
 
-```console
+
+
+```
 sudo hdiutil attach Docker.dmg
 sudo /Volumes/Docker/Docker.app/Contents/MacOS/install
 sudo hdiutil detach /Volumes/Docker
```

Comment:
The better fix for this is to make sure each line has a prompt (the `console` formatting treats lines without a prompt (`$`) as process output, which is not selectable)

let me open a quick PR to address that

Reply:
opened https://github.com/docker/docker.github.io/pull/14543